### ISSUE TITLE
Can i use RNNTLoss in transducer_np.py while training transducer-model?

### ISSUE THREAD

**Issue by stephen-song** [HawkAaron/warp-transducer#19]

**Comment by stephen-song** [comment]
hi, Mingkun , if i dont want to register op  to pytorch, can i import   RNNTLoss from transducer_np.py as my network's loss directly, will it work in training stage?

**Comment by HawkAaron** [comment]
@stephen-song Sure, but its pretty slow, the C++ version is much faster.

**Comment by stephen-song** [comment]
okay ,i see . thx for reply !

**Comment by stephen-song** [comment]
hi , Mingkun , can i just implement  forward_pass to get RNNT loss and let pytorch compute gradient automatically (rather then do it myself)during training ?

**Comment by stephen-song** [comment]
i mean， if i change **np.xx** to **torch.xx** in **transducer_np.py** and only use forward_pass() to get loss and use loss.backward() to automatically apply gradients to weights, will it work? THX in advance !

**Comment by HawkAaron** [comment]
forward_pass contains many inplace operations, like `alpha[t] = alpha[t-1] + ...`, it'll modify the tensor inplace, the gradient cannot pass through it.
